documentation/new-joiners.md

```diff
@@ -0,0 +1,13 @@
+# New joiners checklist
+
+This is a checklist for the on-boarder to run through when a new developer or DevOps joins the team. It may not be exhaustive.
+
+- [ ] Invite to GitHub organization DfE-Digital and team Teacher Vacancy Service.
+- [ ] Give access to:
```

Comment:
Shouldn't this be given automatically by adding the user to the team Teacher Vacancy Service? If not we should probably fix that

Reply:
It is automatic, you're right